Project.toml

```diff
@@ -1,9 +1,10 @@
 name = "KLDivergences"
 uuid = "3c9cd921-3d3f-41e2-830c-e020174918cc"
 authors = ["Theo Galy-Fajou <theo.galyfajou@gmail.com> and contributors"]
-version = "0.2.0"
+version = "0.2.1"
 
 [deps]
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
```

Comment:
You also need to add a compat bound!

Reply:
done. why are there so many manual little steps to remember? :see_no_evil: